src/test/javascript/spec/component/modeling-exercise/modeling-exercise-detail.component.spec.ts

```diff
@@ -47,7 +50,12 @@ describe('ModelingExercise Management Detail Component', () => {
         TestBed.configureTestingModule({
             imports: [ArtemisTestModule],
             declarations: [ModelingExerciseDetailComponent, MockComponent(NonProgrammingExerciseDetailCommonActionsComponent)],
-            providers: [{ provide: ActivatedRoute, useValue: route }, MockProvider(TranslateService)],
+            providers: [
+                { provide: LocalStorageService, useClass: MockSyncStorage },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
+                { provide: TranslateService, useClass: MockTranslateService },
+                { provide: ActivatedRoute, useValue: route },
```

Comment:
Would it make sense to mock the ProfileService here as well?

Reply:
I thought it at first. since test-suit is initialized successfully, I do not think it is necessary. Also, ngOnInit is invoked in the test case, if it is needed it should complain about that